### ISSUE TITLE
[TF2] All health and ammo packs must be at least 13 Hammer Units above the ground to prevent griefing by the Engineers

### ISSUE THREAD

**Issue by KimmyTF2** [ValveSoftware/Source-1-Games#6233]
Friendly players can easily go through the Engineer's buildings, except one. Teleporters have collision enabled for both allies and enemies. This causes a problem when health and ammo packs are at 12 Hammer Units or less above the ground. Teleporters placed directly on them prevent players from picking them up. Enemies won't have a problem with it, but allies won't be able to do anything as long as the Teleport is placed there.
This would require a bit of work to check all maps and raise all pick-ups to be at least 13 Hammer Units above the ground.

https://github.com/user-attachments/assets/eb58a543-1c0e-40f5-9298-608a51b44479

**Comment by JoriKos** [comment]
It might just be easier to extend the pickups hitbox to be taller than go to each individual map and change them

**Comment by KimmyTF2** [comment]
> It might just be easier to extend the pickups hitbox to be taller than go to each individual map and change them

Yeah that actually sounds better. Maybe something like Player Destruction pickups? I've tested them with teleporters and their radius is much bigger.

**Comment by CaptainGame17** [comment]
barely related but MVM money pickups can get stuck in teleporters too

**Comment by gdfsgdfg** [comment]
Oh yes this has bugged me for too long, teleporters blocking pickups should not be a thing at all. Rather fix the object rather than compiling a hundred thousand maps in the game.

**Comment by NULLYUKI** [comment]
> barely related but MVM money pickups can get stuck in teleporters too

According to patch [230553](https://www.teamfortress.com/post.php?id=230553)

> Fixed not being able to pickup currency in MvM if it's on a teleporter 

Please retest this issue.

**Comment by CaptainGame17** [comment]
the issue has been resolved, all pickups are now taller than the teleporter

**Comment by mtxfellen** [comment]
> the issue has been resolved, all pickups are now taller than the teleporter

This does not appear to be the case.
![image](https://github.com/user-attachments/assets/bc921e0c-49a5-4ca8-9516-283c6527aef3)

**Comment by KimmyTF2** [comment]
> the issue has been resolved, all pickups are now taller than the teleporter

Nope. Only MvM money looks to be fixed.

https://github.com/user-attachments/assets/d408a0f2-625b-43aa-93a7-7af440b8b8a9

https://github.com/user-attachments/assets/bf9cf3cb-24ba-4197-b04f-e88935bb7dbc
